pkgs/applications/terminal-emulators/roxterm/default.nix

```diff
@@ -6,25 +6,15 @@
 
 stdenv.mkDerivation rec {
   pname = "roxterm";
-  version = "3.7.5";
+  version = "3.11.1";
 
   src = fetchFromGitHub {
     owner = "realh";
     repo = "roxterm";
     rev = version;
-    sha256 = "042hchvgk9jzz035zsgnfhh8105zvspbzz6b78waylsdlgqn0pp1";
+    sha256 = "1n7588bl83sp51jwjq97f526c7fkh0kq90idw3nayb4zmi530irx";
   };
 
-  patches = [
-    # This is the commit directly after v3.7.5.  It is needed to get roxterm to
-    # build correctly.  It can be removed when v3.7.6 (or v3.8.0) has been
-    # released.
-    (fetchpatch {
```

Comment:
It will be nice to remove `fetchpatch` from args (line 1) as it is no longer used.